ROADMAP.md

```diff
@@ -298,14 +300,14 @@ Status | Command | Description
 
 Status | Command | Description
 ---|--------|------------------------------
-:warning:   | "{char}	        | use register {char} for the next delete, yank, or put
+:warning: :red_circle:  | "{char}	        | use register {char} for the next delete, yank, or put
```

Comment:
@rebornix I think this is compele. But there are other low priority read only registers like command line register that are not implemented.

I've left some comments in register.ts file, I will send PR updating roadmap and adding them as low priority.